src/kanin.app.src

```diff
@@ -1,41 +1,23 @@
-%% -*- erlang -*-
 {application, 'kanin',
- [
-  %% A quick description of the application.
-  {description, "My project description..."},
-
-  %% The version of the application
+ [{description, "An LFE Wrapper for the Erlang RabbitMQ (AMQP) Client"},
   {vsn, "0.3.1"},
-
-  %% All modules used by the application.
   {modules,
-   [
-    'kanin',
+   ['kanin',
     'kanin-chan',
     'kanin-conn',
     'kanin-uri',
-    'kanin-util'
-   ]},
+    'kanin-util']},
 
-  %% All of the registered names the application uses. This can be ignored.
   {registered, []},
-
-  %% Applications that are to be started prior to this one. This can be ignored
-  %% leave it alone unless you understand it well and let the .rel files in
-  %% your release handle this.
+  {included_applications, []},
+  {env, []},
   {applications,
-   [
-    kernel,
-    stdlib
+   [kernel,
+    stdlib,
+    amqp_client,
+    rabbit_common
    ]},
-
-  %% OTP application loader will load, but not start, included apps. Again
-  %% this can be ignored as well.  To load but not start an application it
-  %% is easier to include it in the .rel file followed by the atom 'none'
-  {included_applications, []},
-
-  %% configuration parameters similar to those in the config file specified
-  %% on the command line. can be fetched with gas:get_env
-  {env, []}
- ]
-}.
+  {maintainers, ["LFEx"]},
+  {licenses, [""]},
```

Comment:
Do you mean maintainers and license? LFEx is fine for maintainers ... license should be the standard one that Erlang is moving to -- the Apache 2.0 License (previously discussed with Billo Systems).